src/types.rs

```diff
@@ -153,6 +153,7 @@ pub enum JoinType {
 pub struct OrderExpr {
     pub(crate) expr: SimpleExpr,
     pub(crate) order: Order,
+    pub(crate) nulls_last: Option<bool>,
```

Comment:
It would be better to have an enum named `OrderNull` / `NullOrder` or `Nulls` with variants `First` and `Last` to represents it.